inc/Engine/License/API/User.php

```diff
@@ -72,13 +72,17 @@ public function get_creation_date() {
 	/**
 	 * Checks if user has auto-renew enabled
 	 *
-	 * @return boolean
+	 * @return mixed
 	 */
 	public function is_auto_renew() {
 		if ( ! isset( $this->user->has_auto_renew ) ) {
 			return false;
 		}
 
+		if ( '' === $this->user->has_auto_renew ) {
```

Comment:
Is this an actual use case that can happen? Can our API return an empty value instead of true/false?

Reply:
For me this is something that should be fixed on the API side, it should always return a boolean, not an empty string

Reply:
@Tabrisrp @GeekPress Currently `has_auto_renew` and `has_one-com_account` are returning empty values:
```
"has_auto_renew":"","has_one-com_account":""
```
The same for `is_blocked`.